package.json

```diff
@@ -105,6 +105,7 @@
     "element-closest-polyfill": "1.0.0",
     "extract-text-webpack-plugin": "3.0.2",
     "file-loader": "3.0.1",
+    "file-type": "12.4.0",
```

Comment:
this seems like it'd be a `dependency` not a `devDependency`

Reply:
file-type is only used in the storybook to show an example of a more extensive test for file-type checking after the preliminary checking done in the component.

Reply:
oh ok